service/resource/deployment/types.go

```diff
@@ -25,12 +25,12 @@ type nodeOSImage struct {
 	Version string `json:"version" yaml:"version"`
 }
 
-// newNodeOSImage provides OS information CoreOS 1465.7.0.
-func newNodeOSImageCoreOS_1465_7_0() nodeOSImage {
+// newNodeOSImage provides OS information CoreOS 1576.5.0.
+func newNodeOSImageCoreOS_1576_5_0() nodeOSImage {
```

Comment:
For me this function shoud take as argument channel and version and output proper struct. But now it's hardcoded smth.

Reply:
I think this answers this comment: https://github.com/giantswarm/azure-operator/pull/99#discussion_r160970150